include/benchmark/benchmark.h

```diff
@@ -1676,10 +1675,7 @@ class ConsoleReporter : public BenchmarkReporter {
     OO_Defaults = OO_ColorTabular
   };
   explicit ConsoleReporter(OutputOptions opts_ = OO_Defaults)
-      : output_options_(opts_),
```

Comment:
I'm slightly suspicious of this change ... it's making these fields un-initted, no?
(I've just had to debug a problem very recently when I "forgot" to init the field and assume it'd be null/zero-initialised)

Reply:
it depends on the type being constructed: https://abseil.io/tips/146

the field that's being removed here is `prev_counters_` which is a `UserCounters`, or `std::map<>`.  this has a user-provided (non-default) constructor and so will be value-initialized and safe to use.

however, you're right to be suspicious in general and this is extremely subtle.

Reply:
Ah, I see. That's really subtle! But then, it's C++ - i should expect nothing less! 🙃